src/systems/ec/implementations/vanilla/packages/ec-component-site-header-core/ec-component-site-header-core.scss

```diff
@@ -205,10 +211,21 @@
 
     .ecl-site-header-core__login-toggle,
     .ecl-site-header-core__language-icon {
-      .ecl-icon {
-        height: $ecl-icon-m;
+      .ecl-icon--m {
         margin-right: $ecl-spacing-xs;
-        width: $ecl-icon-m;
+      }
+    }
+
+    .ecl-site-header-core__login-toggle {
+      .ecl-site-header-core__login-icon {
+        display: inline-block;
```

Comment:
`.ecl-site-header-core__language-selector` and `.ecl-site-header-core__login-toggle` are `display: flex;` so I think this should be `display: inline-flex` but please double-check.

Reply:
All headers icons are displayed as `block`, therefore I updated in this way.